src/components/Select.vue

```diff
@@ -1112,10 +1111,14 @@
        * @return {array}
        */
       filteredOptions() {
-        if (!this.filterable && !this.taggable) {
-          return this.mutableOptions.slice()
+        let options = [...this.mutableOptions];
```

Comment:
This makes a shallow copy which I assume is what the `this.mutableOptions.slice()` was doing before.